source/run-command/run-command.rst

```diff
@@ -187,7 +193,160 @@ Drivers MUST document the behavior of RunCommand if a ``maxTimeMS`` field  is al
+     */
+    batchSize?: int32;
+    /**
+     * User configurable time limit enforced by the server.
+     */
+    maxTimeMS?: int32;
+    /**
+     * User configurable comment that can be used to identify the operation in logs.
+     * This can be any BSON value.
+     */
+    comment?: BSONValue;
+  }
+
+The driver's cursor MUST update its ``id`` and ``ns``, as well as store the ``nextBatch`` from every ``getMore`` response.
+
+* See Find, getMore and killCursors commands' section on `GetMore <https://github.com/mongodb/specifications/blob/master/source/find_getmore_killcursors_commands.rst#getmore>`_
+
+Tailable and TailableAwait
+""""""""""""""""""""""""""
+
+By default most cursors are non-tailable, for example, a ``find`` that exhausts when all results for the filter have been satisfied.
+MongoDB also supports creating cursors that "tail" or follow the target namespace for new data.
+Querying capped collections and change streams are some examples of tailable cursor use cases.
+A tailable cursor can receive ``getMore`` responses with an empty ``nextBatch`` array, this does not indicate that the cursor has been exhausted.
+
+In addition to considering a cursor tailable, an ``awaitData`` flag may be sent on the initial command.
+This will request that the server block responding to the ``getMore`` immediately and instead rely on the ``maxTimeMS`` field of the ``getMore`` (or server default).
+If the time does expire an empty batch will be returned and the driver MUST issue another ``getMore``.
+
+It is up to the user to construct their initial command with ``awaitData`` and ``tailable`` flags as well as inform RunCursorCommand of the ``cursorType`` that should be constructed.
+Requesting a ``cursorType`` that does not align with the fields sent to the server on the initial command SHOULD be documented as undefined behavior.
+
+Resource Cleanup
+""""""""""""""""
+
+Drivers MUST provide an explicit mechanism for releasing the cursor resources, typically a ``.close()`` method.
+If the cursor id is nonzero a KillCursors operation MUST be attempted, the result of the operation SHOULD be ignored.
+The ClientSession associated with the cursor MUST be ended and the ServerSession returned to the pool.
+
+* See Driver Sessions' section on `When sending a killCursors command <https://github.com/mongodb/specifications/blob/master/source/sessions/driver-sessions.rst#when-sending-a-killcursors-command>`_
+* See Find, getMore and killCursors commands' section on `killCursors <https://github.com/mongodb/specifications/blob/master/source/find_getmore_killcursors_commands.rst#killcursors>`_
+
+Client Side Operations Timeout
+""""""""""""""""""""""""""""""
+
+As with RunCommand if the initial command has a ``maxTimeMS`` and CSOT settings are provided the behavior is undefined and drivers MUST NOT inspect the document to detect this.
+Drivers SHOULD not allow both a ``getMore`` ``maxTimeMS`` setting to be provided and CSOT settings.
```

Comment:
can you rephrase this sentence? it's unclear what exactly it's trying to say, is it ```Drivers SHOULD *NOT* allow both a ``getMore`` *and a* ``maxTimeMS`` setting to be provided *in* CSOT settings.```?

Reply:
Flipped the boolean here, saying we _should_ raise an error, I predict this could be difficult for some drivers (node.js) depending on when they receive CSOT options. The documentation requirement is the important part.